catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -266,9 +335,9 @@ private RemoteRegistryResults getRemoteRegistryMetacardsMap() throws FederationA
                 try {
                     results.add(future.get(taskWaitTimeSeconds, TimeUnit.SECONDS));
                 } catch (ExecutionException e) {
-                    LOGGER.debug("Error executing query on a remote registry.", e);
+                    LOGGER.error("Error executing query on a remote registry.", e);
```

Comment:
Anything at warn or above needs to tell an administrator what they can do to resolve the issue.